core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -168,6 +168,15 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
   }
 
   /** Set multiple parameters together */
+  def setAll(settings: Iterable[(String, String)]): SparkConf = {
```

Comment:
Any caller is almost certainly passing an object that is `Iterable` as well as `Traversable`, but this maintains binary compatibility for now.